spring-cloud-huawei-swagger/src/main/java/com/huaweicloud/swagger/ServiceCombSwaggerHandlerImpl.java

```diff
@@ -113,19 +114,18 @@ public void init(String appName, String serviceName) {
           }
         });
         if (withJavaChassis) {
-          filteSwagger(temSwagger, fullClassName);
+          filterSwagger(temSwagger, fullClassName);
           if (CollectionUtils.isEmpty(temSwagger.getPaths())) {
             continue;
           }
           //add text/plain for string
           List contentTypeList = Arrays.asList("text/plain", "application/json");
-          temSwagger.getPaths().forEach((path, operation) -> {
-            operation.getOperations().forEach(api -> {
-              api.setConsumes(contentTypeList);
-              api.setProduces(contentTypeList);
-              api.setTags(Arrays.asList(className));
-            });
-          });
+          temSwagger.getPaths().forEach((path, operation) ->
+              operation.getOperations().forEach(api -> {
+                api.setConsumes(contentTypeList);
+                api.setProduces(contentTypeList);
+                api.setTags(Arrays.asList(className));
+              }));
```

Comment:
It seems not change. Is it necessary to delete {} ?

Reply:
Idea suggests to delete { } for single statement in anonymous functions to simplify code block. The { } in lambda from lambda is not as meaningful as those in condition or loop statements. However, I think either is fine in this situation.

Reply:
remove {} and don't put multiple loops on one line, seems easier to read

Reply:
done